src/test/java/org/folio/linked/data/e2e/resource/ResourceControllerITBase.java

```diff
@@ -376,6 +384,17 @@ void createWorkWithInstanceRef_shouldSaveEntityCorrectly() throws Exception {
   @Test
   void update_shouldReturnCorrectlyUpdatedInstanceWithWorkRef_deleteOldOne_sendMessages() throws Exception {
     // given
+    var specifications = new SpecificationDtoCollection();
+    var specRuleId = randomUUID();
+    specifications.setSpecifications(List.of(new SpecificationDto().id(specRuleId)));
+    var specRules = new SpecificationRuleDtoCollection();
+    var specRule = new SpecificationRuleDto();
+    specRule.setCode("invalidLccnSubfieldValue");
+    specRule.setEnabled(true);
+    specRules.setRules(List.of(specRule));
```

Comment:
i guess we can extract data preparation to separate method to make the test shorter

Reply:
done